README.md

```diff
@@ -1265,7 +1276,37 @@ Example result:
 
 ```
 {'defaultAggregation': 'AVG', 'metric': 'nvt'}
+
 ```
+
+## Extras
+
+### Backtest
+
+TBD
```

Comment:
Shouldn't we add some info here?

Reply:
lets add info or remove from README

Reply:
Removed the backtest section and left the extras section with a link to examples folder.